core/src/test/scala/unit/kafka/server/TopicIdWithOldInterBrokerProtocolTest.scala

```diff
@@ -15,12 +15,13 @@
  * limitations under the License.
  */
 
-package kafka.server
+package unit.kafka.server
```

Comment:
We use `kafka.server` for the other unit tests in this directory, so we should do the same here (in Scala, the package name doesn't need to match the directory structure).

Reply:
Got it.